pkg/handlers/authorization/handler.go

```diff
@@ -79,6 +80,7 @@ func (auth *authorizationHandler) Process(req *http.Request, context *handlers.R
 			}
 		}
 		req.Header.Add(headerForwardedRoles, strings.Join(context.RoleSet().List(), ","))
+		req.Header.Add(headerForwardedFor, "::1")
```

Comment:
Why only for the token case and not for the subject extractor case?

Reply:
yes.  This needs to be added for any traffic from the proxy to ES

Reply:
in the 'subject extractor' case other headers are missing as well (x-forwarded-roles, x-ocp-ns etc).

Reply:
this is fine because we are deferring those roles to the ones defined in the rolesmapping for the server